app/models/user.py

```diff
@@ -47,12 +47,14 @@ class User(JSONModel, UserMixin):
         "password_changed_at",
         "permissions",
         "state",
+        "preferred_timezone",
     }
 
     def __init__(self, _dict):
         super().__init__(_dict)
         self.permissions = _dict.get("permissions", {})
         self._platform_admin = _dict["platform_admin"]
+        self.preferred_timezone = "US/Eastern"
```

Comment:
I'm assuming this is what will be updated afterward to actually use the user's preference now that the API change is in?

Reply:
presumably!   But the user can't actually specify their preference until a widget exists in the UI, there's a separate ticket for that.